filebeat/module/elasticsearch/deprecation/test/elasticsearch_deprecation.log-expected.json

```diff
@@ -2,7 +2,7 @@
     {
         "@timestamp": "2018-04-23T16:40:13.737Z",
         "ecs.version": "1.0.0-beta2",
-        "elasticsearch.server.component": "o.e.d.a.a.i.t.p.PutIndexTemplateRequest",
+        "elasticsearch.component": "o.e.d.a.a.i.t.p.PutIndexTemplateRequest",
```

Comment:
Breaking change or never released?

Reply:
Breaking change. Felt odd to me that the `deprecation` fileset was writing into  the `server` fileset's namespace. So instead I decided to promote the `component` field up to their common ancestor.

I'll add an additional CHANGELOG entry for this breaking change.